x-pack/plugins/apm/server/routes/services/get_services/get_service_aggregated_transaction_stats.ts

```diff
@@ -151,6 +157,7 @@ export async function getServiceAggregatedTransactionStats({
           end,
           value: topTransactionTypeBucket.doc_count,
         }),
+        overflowCount: bucket.overflowCount.value,
```

Comment:
Not related to this PR but I was struggling to figure out the difference between `getServiceAggregatedTransactionStats` and `getServiceTransactionStats`. It looks like `getServiceAggregatedTransactionStats` is for service metrics, and `getServiceTransactionStats` is for transaction metrics and transaction events. 

Can you change the naming so that is more clear, eg

`getServiceAggregatedTransactionStats` -> `getServiceStatsForServiceMetrics`
`getServiceTransactionStats` -> `getServiceStats`

cc @kpatticha